app/controllers/miq_policy_controller.rb

```diff
@@ -922,20 +941,21 @@ def folder_get_info(folder_node)
         @sb[:folder] = "#{nodeid.split("-")[1]}-#{nodeid.split("-")[2]}"
         set_search_text
         policy_get_all if folder_node.split("_").length <= 2
-        @right_cell_text = _("All %{typ} %{model}") % {:typ => ui_lookup(:model => @sb[:nodeid]), :model => ui_lookup(:models => "MiqPolicy")}
+        @right_cell_text = _("All %{typ} %{model}") % {:typ   => ui_model_from_id(@sb[:nodeid]),
+                                                       :model => ui_lookup(:models => "MiqPolicy")}
         @right_cell_div = "policy_list"
       end
     elsif x_active_tree == :condition_tree
       @conditions = Condition.where(:towhat => @sb[:folder].camelize).sort_by { |c| c.description.downcase }
       set_search_text
       @conditions = apply_search_filter(@search_text, @conditions) unless @search_text.blank?
-      @right_cell_text = _("All %{typ} Conditions") % {:typ => ui_lookup(:model => @sb[:folder])}
+      @right_cell_text = _("All %{typ} Conditions") % {:typ => ui_model_from_id(@sb[:folder])}
       @right_cell_div = "condition_list"
     elsif x_active_tree == :alert_profile_tree
       @alert_profiles = MiqAlertSet.where(:mode => @sb[:folder]).sort_by { |as| as.description.downcase }
       set_search_text
       @alert_profiles = apply_search_filter(@search_text, @alert_profiles) unless @search_text.blank?
-      @right_cell_text = _("All %{typ} Alert Profiles") % {:typ => ui_lookup(:model => @sb[:folder])}
+      @right_cell_text = _("All %{typ} Alert Profiles") % {:typ => ui_model_from_id(@sb[:folder])}
```

Comment:
This is not strictly necessary as ids in Alert Profiles have well-camelized model: `ap_xx-EmsCluster_ap-21_al-22`.
(either lookup works for `@sb[:folder] == "EmsCluster"` and fails for `@sb[:folder] == "ap-21"; the latter is not user-visible as right_cell_text is overridden by another method.)

Including this change since it's the only place with well-camelized models in ids, `ui_model_from_id` is appropriate for the intent here, and doing the camelize anyway is a no-op.